README.md

```diff
@@ -151,12 +151,24 @@ Packages that exist in the original libp2p specs and are under active developmen
 
 ### Tips and tricks
 
-- enable expensive metrics:
+#### enable expensive metrics:
 
 ```bash
 nim c -d:libp2p_expensive_metrics some_file.nim
 ```
 
+#### use identify metrics
+
+```bash
+nim c -d:libp2p_agents_metrics -d:KnownLibP2PAgents=nimbus,lighthouse,prysm,teku some_file.nim
```

Comment:
Is there any way of enabling this metrics at runtime? I'm not sure if they are all compile time, but maybe something like what NBC does with runtime log levels would be useful?

Reply:
we can add an RPC endpoint to NBC to toggle those.

Reply:
that is true, I will just make them a `var` rather then `const`